internal/e2e/core/cluster_node.go

```diff
@@ -54,7 +55,8 @@ type StringSlice []string
 type WorkloadSlice []*Workload
 
 type Workload struct {
-	ID string
+	ID    string
+	Value int
```

Comment:
While I was here, I also edited the e2e to  have an example of int-to-int32 conversion which does require a user provided function since it could be lossy.